_pages/dev/tms.md

```diff
@@ -0,0 +1,205 @@
+# Tag Management System
+
+Spartacus introduced the Tag Management System (TMS) integration in version 3.2.
+Before you continue reading this page, it's highly recommended to first get yourself familiar with the [Events Service](./event-service), as TMS relies on it.
+
+The TMS integration allows you to specify which Spartacus' events should be passed to the configured TMS. Google Tag Manager (GTM) and Adobe Experience Platform (AEP) systems are supported out of the box, while the other tag managers can easily be plugged in.
```

Comment:
> Adobe Experience Platform

I believe it's Adobe Experience Launch Platform, as the AE covers a wide range of products

Reply:
Done. 
Should we also reflect this change in the code?
`aep` -> `aelp` as the configuration name, etc.